CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## Unreleased
+
+## Added
+
+- TLS support
+
+  `--tls` option enables TLS and uses system CA certs for host verification
+  `--tls-cert` option specifies certificate to use for host verification
```

Comment:
I may have messed with terminology here. `--tls-cert` specifies server certificate. I planned this to be used for self-signed certificates.

Reply:
I don't think it's practical to use server certificate here. The usual practice is to have a self-signed CA that is being trusted via such a command.  Given that it's also a common practice to have an intermediate CA between root and particular server being able to specify a dir to loads certs becomes even more important so that it will be possible to build a trust chain.